server/apps/app.go

```diff
@@ -42,68 +44,16 @@ type App struct {
 	GrantedLocations Locations `json:"granted_locations,omitempty"`
 }
 
-func (m *Manifest) ConfigMap() map[string]interface{} {
-	return map[string]interface{}{
-		"app_id":                string(m.AppID),
-		"display_name":          m.DisplayName,
-		"description":           m.Description,
-		"oauth2_callback_url":   m.OAuth2CallbackURL,
-		"homepage_url":          m.HomepageURL,
-		"root_url":              m.RootURL,
-		"requested_permissions": m.RequestedPermissions.toStringArray(),
-		"requested_locations":   m.RequestedLocations.toStringArray(),
-	}
-}
-
-func ManifestFromConfigMap(in interface{}) *Manifest {
-	m := &Manifest{}
-	c, _ := in.(map[string]interface{})
-	if len(c) == 0 {
-		return m
-	}
-	appID, _ := c["app_id"].(string)
-	m.AppID = AppID(appID)
-	m.DisplayName, _ = c["display_name"].(string)
-	m.Description, _ = c["description"].(string)
-	m.OAuth2CallbackURL, _ = c["oauth2_callback_url"].(string)
-	m.HomepageURL, _ = c["homepage_url"].(string)
-	m.RootURL, _ = c["root_url"].(string)
-	m.RequestedPermissions = permissionsFromConfigArray(c["requested_permissions"])
-	m.RequestedLocations = locationsFromConfigArray(c["requested_locations"])
-	return m
-}
-
 func (a *App) ConfigMap() map[string]interface{} {
-	return map[string]interface{}{
-		"manifest":             a.Manifest.ConfigMap(),
-		"secret":               a.Secret,
-		"oauth2_client_id":     a.OAuth2ClientID,
-		"oauth2_client_secret": a.OAuth2ClientSecret,
-		"oauth2_trusted_app":   a.OAuth2TrustedApp,
-		"bot_user_id":          a.BotUserID,
-		"bot_username":         a.BotUsername,
-		"bot_access_token":     a.BotAccessToken,
-		"granted_permissions":  a.GrantedPermissions.toStringArray(),
-		"granted_locations":    a.GrantedLocations.toStringArray(),
-	}
+	b, _ := json.Marshal(a)
```

Comment:
There's no reason for these functions then, is there? At least not ToMap - it can be generic in config.store

Reply:
If we want to keep the ConfigMapper for future use or in case we change anything, we can keep them (I do not foresee any case we will need it, but I haven't thought much about it). If not, we can let the mapping be done directly by Store function.